### ISSUE TITLE
Cannot adjust enabled cryptocurrency coin types from current mojo methods

### ISSUE THREAD

**Issue by josheleonard** [brave/brave-browser#33775]
## Description 

We will need a way to enable or disable SOL, FIL, and ETH features of the wallet from a mojo method to allow the user to decide which account types they would like to have created when creating a wallet. We will Also need to start tracking if ETH is enabled or not.

## Steps to Reproduce

   1. open brave://flags
   2. search for Eth or EVM

## Actual result:

No option to disable Eth or EVM features, only SOL and FIL

## Expected result:
Show an option to disable Eth or EVM features

## Version/Channel Information:

- Can you reproduce this issue with the current release? 
- Can you reproduce this issue with the beta channel? 
- Can you reproduce this issue with the nightly channel? 

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome?

**Comment by yrliou** [comment]
I don't think we would want a feature flag for ETH/EVM, on the contrary, we should retire solana, filecoin feature flags as they're now shipped and mature on all the platforms.
